src/main.js

```diff
@@ -75,16 +85,23 @@ export const createFormReducer = formConfig => (
       
       return produce(state, draftState => {
         draftState[fieldWithOverride].validators.push(newValidator)
-        const fields = Object.entries(draftState);
-        for (let entry of fields) {
-          let fieldName = entry[0];
-          let field = entry[1];
-          let errors = computeErrors(fieldName, draftState);
-          let dirty = field.dirty;
-          draftState[fieldName].errors = errors;
-          draftState[fieldName].dirty = dirty;
-          draftState[fieldName].hasErrors = errors.length > 0;
-        }
+        computeErrorEntries(draftState);
+      });
+    case REMOVE_VALIDATOR:
+      const fieldToOverride = action.payload.fieldName;
+      const targetValidator = action.payload.validator;
+
+      return produce(state, draftState => {
+        let fieldValidators = draftState[fieldToOverride].validators
+        draftState[fieldToOverride].validators = fieldValidators.filter(validator => validator.type !== targetValidator.type)
+        computeErrorEntries(draftState);
+      });
+    case CLEAR_FIELD_VALIDATORS:
```

Comment:
This is 100% fine as is and a great addition, but if the lift to remove only one `type` of validator from a field might be useful too. Do not let this suggestion block this work though; if it's an easy lift it might be worth including here, but not otherwise I think.